src/object/srv_obj.c

```diff
@@ -1898,6 +1898,65 @@ ds_obj_tgt_update_handler(crt_rpc_t *rpc)
 	obj_ioc_end(&ioc, rc);
 }
 
+static int
+obj_dth_agg_cb(struct dtx_leader_handle *dlh, void *agg_arg)
+{
+	uint64_t	*flag = agg_arg;
+	int		succeeds = 0;
+	int		allow_failure = 0;
+	int		allow_failure_cnt = 0;
+	int		result = 0;
+	int		i;
+
+	D_ASSERT(flag != NULL);
+	if (*flag && (DAOS_COND_PUNCH | DAOS_COND_DKEY_UPDATE |
+		     DAOS_COND_AKEY_UPDATE))
+		allow_failure = -DER_NONEXIST;
+	else if (*flag && (DAOS_COND_AKEY_INSERT | DAOS_COND_DKEY_INSERT))
```

Comment:
should this be a & rather than a &&?   Same for 1912

Reply:
ah. right.